Migrations/20201023215527_InitialCreate.cs

```diff
@@ -0,0 +1,513 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace InvestingOak.Migrations
+{
+    public partial class InitialCreate : Migration
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6393445)